src/iconv.jl

```diff
@@ -1,20 +1,48 @@
 # This file is a part of Julia. License is MIT: http://julialang.org/license
 
 module iconv
-import Base: close, eof, flush, read, readall, write
-import Base.Libc: errno, strerror, E2BIG, EINVAL, EILSEQ
-export StringEncoder, StringDecoder, encode, decode
+import Base: close, eof, flush, read, readall, write, show
+import Base.Libc: errno, E2BIG, EINVAL, EILSEQ
+export StringEncoder, StringDecoder, encode, decode, StringErr, StringError
+
+module StringErr
+const global UNKNOWN_ERR      = "<<1>>: <<2>> (<<3>>)"
+const global ICONV_INSTALL    = "libiconv not properly installed. Please run\nPkg.build(\"iconv\")"
```

Comment:
I wouldn't make this the same exception type as other errors. It will only happen when loading the package, I can be a standard `ErrorException`. Ideally, `BinDeps` would provide a standardize way of checking this, with a specific error.